### PR TITLE
HttpClient is explicit dependency now, so user can configure it

### PR DESCRIPTION
**Pull request by Neftedollar**
HttpClient was created implicitly and it was impossible to configure it(for example it was impossible to set proxy)
Now HttpClient is a part of config, it's ok cause HttpClient [is a collection of settings](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=netframework-4.7.2#remarks) too.
Now it explicit, but does not require unnecessary attention.

### CODE REVIEW

File: Funogram/Bot/Bot.fs [hunk 1]
```diff
@@ -99,7 +103,7 @@ let private runBot config me updateArrived updatesArrived =
 let startBot config updateArrived updatesArrived = 
     let meResult = 
         getMe
-        |> api config.Token
+        |> api config.Client config.Token
```
Comment (worldbeater) [inline Funogram/Bot/Bot.fs]: This seems to be a breaking change in API. Before we've used this code to invoke a Telegram Bot API method:
```fsharp
"Sample message"
|> sendMessage message.Chat.Id
|> api context.Config.Token
```

But now it seems we have to use this more verbose version:
```fsharp
"Sample message"
|> sendMessage message.Chat.Id
|> api context.Config.Client context.Config.Token
```

Maybe the `api` function should accept a config instance and extract parameters internally? 
```
"Sample message"
|> sendMessage message.Chat.Id
|> api context.Config
```

Comment (Neftedollar) [reply Funogram/Bot/Bot.fs]: Я тоже подумал, что логичнее было бы принимать, конфиг.

File: Funogram.TestBot/Program.fs [hunk 2]
```diff
@@ -1,110 +1,123 @@
...
+                ))
+                cmd "/forward_message" (fun _ -> bot (forwardMessage (fromId()) (fromId()) ctx.Update.Message.Value.MessageId))
+                cmd "/show_my_photos_sizes" (fun _ -> 
+                (
+                    let x = botResult (getUserProfilePhotosAll (fromId())) |> processResultWithValue
+                    if x.IsNone then ()
+                    else
+                        let text = sprintf "Photos: %s" (x.Value.Photos 
+                                    |> Seq.map (Seq.last >> (fun f -> sprintf "%ix%i" f.Width f.Height))
+                                    //|> Seq.map (fun f -> sprintf "%ix%i" f.Width f.Height)
```
Comment (ForNeVeR) [inline Funogram.TestBot/Program.fs]: 😠

Comment (Neftedollar) [reply Funogram.TestBot/Program.fs]: ??

**Pull request merged**